### DISCUSSION TITLE
Turbsim with Aero_driver (GF wind array was exhaustrd at 0.08264s )

### DISCUSSION THREAD

**Discussion by robytian** [OpenFAST/openfast#2550] [Q&A]
Dear all
I tried to simulate two 5MW rotors in the Aerodriver program, and the wheel hub height was placed at 200m. I wanted to simulate the wind area where X*Y was 600*400. My inp was set as follows, the azimuthing Angle was 6°, and RPM12.1, so the time step was 0.08264. But the calculation times incorrectly remind me that I need to double the time step, why?
![2e2cfe820979e57bc5152d228b6031c](https://github.com/user-attachments/assets/82663ed8-9301-40f9-97d5-4308a379d85c)
![734d046fcce7825f2635a9c83593012](https://github.com/user-attachments/assets/10b4f1cb-0a30-415a-8fe3-35d15212538f)
![095377321f252bc9e14901cd55abe51](https://github.com/user-attachments/assets/a3f95982-c798-4afb-a541-8b047869cfa2)

**Comment by robytian** [comment]
I've tried adjusting the time step in .inp to the recommended 0.0324439, and a smaller 0.02, and still get the error.

**Reply by jjonkman** [reply]
Dear @robytian,

I'm not exactly sure why you are receiving this error, but I would suggest changing `UsableTime` in TurbSim to "ALL" and regenerating the wind data file, which will make the TurbSim output periodic in the X / time direction, which should avoid the standalone AeroDyn driver from generating any errors about the wind array not being large enough in the X / time direction.

Best regards,
